es-extension-examples/ext-example-basic/package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "prettier --check \"src/**/*.{js,jsx,ts,tsx}\"",
     "maybe-install": "[ -d node_modules ] && [ -z $DO_INSTALL ] || npm install",
     "container": "npm run maybe-install && vite dev",
-    "container-preview": "npm run maybe-install && vite preview"
+    "container-preview": "npm run maybe-install && npm run build && vite preview --outDir ."
```

Comment:
this potentially exposes everything in the root directory of the sub-project for production/preview